app/models/chargeback_rate.rb

```diff
@@ -11,6 +20,8 @@ class ChargebackRate < ApplicationRecord
   validates_uniqueness_of   :guid
   validates_uniqueness_of   :description, :scope => :rate_type
 
+  virtual_column :assigned?, :type => :boolean
```

Comment:
If the UI wants to show/hide the delete button, I would want the UI to call a method like `can_destroy?` or `valid_destroy?` on the model as in http://stackoverflow.com/questions/123078/how-do-i-validate-on-destroy-in-rails.  That method can encapsulate the `assigned?` method below.  We should not be teaching the UI *why* we cannot destroy something -- that is the model's responsibility.

As for whether `assigned?` is useful in reporting, that is a legitimate, but separate question.